test/http-request.js

```diff
@@ -233,7 +247,7 @@ describe("http-request", function() {
                     throw new Error("ShouldNotBeCalled");
                 })
                 .catch(function(error) {
-                    expect(error).to.match(/HttpConnectionError/);
+                    expect(error).to.match(/HttpConnexionError/);
```

Comment:
Why did you change the matched error? `Connexion` is a French word, so it should not be used as error name.

Reply:
I don't remember editing this consciously. Fixed